src/fuel_fab.cc

```diff
@@ -507,6 +507,13 @@ void FuelFab::RecordPosition() {
       ->Record();
 }
 
+void FuelFab::Tick() {
+  cyclus::toolkit::RecordTimeSeries<double>(fuelfab_record_demand, this, 
+                                            throughput);
```

Comment:
Instead of throughput, wouldn't this be:
1. `fill_size - fill.quantity()`
2. `fiss_size - fiss.quantity()`
Since that is the remaining capacity for accepting?

Reply:
yup. I think the recordtimeseries would be best if it was a vector.